PPSTools/Utilities/interface/PPSOpticsCalibrator.h

```diff
@@ -0,0 +1,94 @@
+#ifndef PPS_OPTICS_CALIBRATION
```

Comment:
instead of creating a new subsystem here i would suggest looking for appropriate spots that already exist. Perhaps Utilities/PPS if there is not something that already exists..

Reply:
Agreed. Implementing it...

Reply:
Done. It was created the module PPS inside Utilities as said above and the corresponding changes propagated to the rest of the code.